packages/utils/tests/getIntegerInRangeFromString.test.ts

```diff
@@ -0,0 +1,40 @@
+import { getIntegerInRangeFromString } from '../src/getIntegerInRangeFromString';
+
+describe('getIntegerInRangeFromString', () => {
+    it('should always produce the same number for the same string input', () => {
+        const input = '47A2176AC04E609FD0E9811A';
+        const result1 = getIntegerInRangeFromString(input, 101);
+        const result2 = getIntegerInRangeFromString(input, 101);
+        expect(result1).toEqual(result2);
+    });
+
+    it('should always be deterministic in producing a number from same input', () => {
+        const input = 'z';
+        const result1 = getIntegerInRangeFromString(input, 101);
+        expect(result1).toBe(21);
+    });
+
+    it('should produce different numbers for different string inputs', () => {
```

Comment:
Well this may not be true in general since there's only a hundred possible outputs. I mean, it's a coincidence (high probability tho) that these different strings produce different numbers.

Reply:
You are right, I removed that test case.